portfolio/src/main/webapp/index.html

```diff
@@ -31,13 +31,22 @@ <h1>Lily Zhou</h1>
         <button class="button"><a href="images.html">Food Journal</a></button>
         <button class="button"><a href="projects.html">Projects</a></button>
       </div>
-      <form action="/data" method="POST">
-        <textarea name="name-input">Name</textarea>
-        <textarea name="comment-input">Your comment here.</textarea>
-        <br/><br/>
-        <input type="submit" />
-      </form>
-      <div id="comments-container"></div>
+      <div class="comments-box">
+        <form action="/data" method="POST">
+          <label for="name-input" class="comment-label">Name: </label><br>
+          <input type="text" name="name-input"><br><br>
+          <label for="comment-input" class="comment-label" rows="4" cols="100">Comment: </label><br>
+          <textarea name="comment-input" class="text-input"></textarea><br><br>
+          <label for="comment-input" class="comment-label">Max Comments Displayed: </label>
```

Comment:
I don't think this should be part of this form.  I think the "max comments" should be part of different section, not the same one that lets you leave a comment.  This should be a request param for your GET request, not the POST request.